applications/standalone/src/mock/context.ts

```diff
@@ -121,7 +121,26 @@ const newMockDataRepositoryTable = (numItems = 30): DataRepositoryItemsTable =>
     type: Utf8Vector.from(isTableType.map(isTable => (isTable ? 'table' : 'string'))),
     columnNames: ListVector.from({
       values: isTableType.map((isTable, idx) =>
-        isTable ? [`a${idx}`, `b${idx}`, `c${idx}`, `d${idx}`, `e${idx}`, `f${idx}`] : null
+        isTable
+          ? [
```

Comment:
I know I started it this way, but with half the alphabet maybe we can do this :) 

```
[...Array(16).keys()].map(i => `${String.fromCharCode(97+i)}${i}`)
```

Reply:
Ha-ha! Good point!